packages/mdk4/build.sh

```diff
@@ -0,0 +1,20 @@
+#MDK is a proof-of-concept tool to exploit common IEEE 802.11 protocol weaknesses.
+# MDK4 is a new version of MDK3.
+
+#MDK4 is a Wi-Fi testing tool from E7mer of 360, ASPj of k2wrlz, it uses the osdep library from the aircrack-ng project to inject frames on several operating systems. Many parts of it have been contributed by the great aircrack-ng community: Antragon, moongray, Ace, Zero_Chaos, Hirte, thefkboss, ducttape, telek0miker, Le_Vert, sorbo, Andy Green, bahathir, Dawid Gajownik and Ruslan Nabioullin. THANK YOU!
+
+#MDK4 is licenced under the GPLv2 or later.
+
+TERMUX_PKG_MAINTAINER="KimoCoder & Ezmer "
+TERMUX_PKG_HOMEPAGE=https://github.com/aircrack-ng/mdk4
+TERMUX_PKG_DESCRIPTION="MDK is a proof-of-concept tool to exploit common IEEE 802.11 protocol weaknesses."
+TERMUX_PKG_LICENSE="GPL-3.0"
+
+
+TERMUX_PKG_VERSION=4.1
+TERMUX_PKG_REVISION=1
```

Comment:
Remove this line, it does not have to be set for new packages

Reply:
without these lines there are build error so kept these lines but turned the value empty

Reply:
TERMUX_PKG_VERSION=4.1 needs to be there. TERMUX_PKG_REVISION=1 can be removed

Reply:
ok got it will do that

Reply:
removing the line TERMUX_PKG_REVISION=1 stops the build process but keeping its value empty dosent affect the build process so I kept it empty and also tested arm and aarch64 both developed correctly